.gitignore

```diff
@@ -128,3 +132,8 @@ dist
 .yarn/build-state.yml
```

Comment:
Its fine to have this, but please use npm rather than yarn to be consistent with rest of jambonz project team